code/index.html

```diff
@@ -1,30 +1,160 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Forum&family=Work+Sans:wght@300&display=swap" rel="stylesheet">
+
+  <title>Earthely - Project 3</title>
+
+</head>
+
+<body>
+
+  <header>
+    <div class="header">
+      <h1 class="hero-title">EARTHELY</h1>
+    </div>
+    <div class="hero-text-box">
+      <h2 class="h2">We are a collective of ordinary people united on a mission to save the planet.</h2>
+      <p class="header-text"> We provide resources and simple tips and tricks to individuals and communities around the
+        world, so that together we can make small and effective changes to show our Mother Earth the love she deserves.
+      </p>
+    </div>
+
+    <div id="hero">
+      <video playsinline autoplay muted loop poster="./videos/placeholder." id="bgvideo" width="100%" height="100%">
+        <source src="https://file-examples-com.github.io/uploads/2017/04/file_example_MP4_1920_18MG.mp4"
+          type="video/mp4">
+      </video>
+
+  </header>
+
+  <section>
+    <div class="none"></div>
+  </section>
+
+  <!-- This is the section for the form and accordion -->
+
+  <section class="two">
+    <div class="form">
+      <form action="http://httpbin.org/anything" method="post" id="form" class="form">
+        <h1>EARTHELY NEWSLETTER</h1>
+        <p>Every week we share simple and powerful resources to help you get involved.</p>
+        <label class="label" for="firstname">First name</label>
+        <input class="name textblock" type="text" id="firstname" name="Firstname" placeholder="First name" required />
+
+        <label id="form" class="label" for="name">Last name</label>
+        <input class="name textblock" type="text" id="lastname" name="Lastname" placeholder="Last name" value=""
+          required />
+
+
+        <label id="form" class="label" for="name">Email Address</label>
+        <input class="name textblock" type="email" id="email" name="email" placeholder="Email" value="" required />
+
+        <br>
+
+        <label id="form" class="label" for="name">Date of birth </label>
+        <input class="name" type="date" id="date" name="date" value="" required />
```

Comment:
For some reason, I can put 6 digits on the year. Maybe set a min max value? ` min="1900-01-01" max="2021-0101` on input for example.

Reply:
Great observation, I have not even tried filling it manually, I just used the calender that pops-up.